library/src/main/scala/g8.scala

```diff
@@ -82,14 +100,21 @@ object G8 {
   }
 
   def write(in: File, out: File, parameters: Map[String, String], append: Boolean) {
-    Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
-      case Some(attr) =>
-        val mode = attr.getOctalMode
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
-        util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
-      case None =>
-        // PlexusIoResourceAttributes is not available for some OS'es such as windows
-        write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+    try {
+      Option(PlexusIoResourceAttributeUtils.getFileAttributes(in)) match {
+        case Some(attr) =>
+          val mode = attr.getOctalMode
+          write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+          util.Try(ArchiveEntryUtils.chmod(out, mode, new ConsoleLogger(Logger.LEVEL_ERROR, "")))
+        case None =>
+            // PlexusIoResourceAttributes is not available for some OS'es such as windows
+            write(out, FileUtils.readFileToString(in, "UTF-8"), parameters, append)
+      }
+    }
+    catch {
+      case t : Throwable =>
+        println("Falling back to file copy for %s: %s" format(in.toString, t.getMessage))
```

Comment:
Just for reference here is what happens now if there is an error in the template:

```
> g8 'file:////Users/njlg/source/tmp/something/my-template-project' --name='Web Services'
32:49: '"
      val status = result.header.status
      val forwardedFor = requestHeader.headers.get("' came as a complete surprise to me
org.stringtemplate.v4.compiler.STException
	at org.stringtemplate.v4.compiler.Compiler.reportMessageAndThrowSTException(Compiler.java:213)
	at org.stringtemplate.v4.compiler.Compiler.compile(Compiler.java:143)
	at org.stringtemplate.v4.STGroup.compile(STGroup.java:448)
	at org.stringtemplate.v4.ST.<init>(ST.java:173)
	at org.clapper.scalasti.STHelper$.apply(ScalastiHelper.scala:8)
	at giter8.G8$.applyTemplate(g8.scala:43)
	at giter8.G8$.write(g8.scala:97)
	at giter8.G8$.write(g8.scala:88)
	at giter8.G8Helpers$$anonfun$write$3$$anonfun$apply$13.apply(g8.scala:325)
	at giter8.G8Helpers$$anonfun$write$3$$anonfun$apply$13.apply(g8.scala:325)
	at scala.util.control.Exception$Catch$$anonfun$opt$1.apply(Exception.scala:118)
	at scala.util.control.Exception$Catch$$anonfun$opt$1.apply(Exception.scala:118)
	at scala.util.control.Exception$Catch.apply(Exception.scala:102)
	at scala.util.control.Exception$Catch.opt(Exception.scala:118)
	at giter8.G8Helpers$$anonfun$write$3.apply(g8.scala:324)
	at giter8.G8Helpers$$anonfun$write$3.apply(g8.scala:303)
	at scala.collection.immutable.Stream.foreach(Stream.scala:547)
	at giter8.G8Helpers$.write(g8.scala:303)
	at giter8.G8Helpers$$anonfun$giter8$G8Helpers$$applyT$1.apply(g8.scala:169)
	at giter8.G8Helpers$$anonfun$giter8$G8Helpers$$applyT$1.apply(g8.scala:158)
	at scala.util.Either$RightProjection.flatMap(Either.scala:523)
	at giter8.G8Helpers$.giter8$G8Helpers$$applyT(g8.scala:158)
	at giter8.G8Helpers$$anonfun$applyTemplate$2.apply(g8.scala:185)
	at giter8.G8Helpers$$anonfun$applyTemplate$2.apply(g8.scala:185)
	at giter8.Apply$$anonfun$inspect$1.apply(apply.scala:51)
	at giter8.Apply$$anonfun$inspect$1.apply(apply.scala:50)
	at scala.util.Either$RightProjection.flatMap(Either.scala:523)
	at giter8.Apply$class.inspect(apply.scala:50)
	at giter8.Giter8.inspect(giter8.scala:3)
	at giter8.Giter8$$anonfun$2.apply(giter8.scala:22)
	at giter8.Giter8$$anonfun$2.apply(giter8.scala:18)
	at scala.Option.map(Option.scala:145)
	at giter8.Giter8.run(giter8.scala:18)
	at giter8.Giter8.run(giter8.scala:10)
	at giter8.Giter8.run(giter8.scala:3)
	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
	at xsbt.boot.Launch$.run(Launch.scala:109)
	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
	at xsbt.boot.Launch$.launch(Launch.scala:117)
	at xsbt.boot.Launch$.apply(Launch.scala:18)
	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
	at xsbt.boot.Boot$.main(Boot.scala:17)
	at xsbt.boot.Boot.main(Boot.scala)
Error during sbt execution: org.stringtemplate.v4.compiler.STException
```